common/ast/src/ast/statement.rs

```diff
@@ -351,6 +374,60 @@ pub enum DatabaseEngine {
     Github(String),
 }
 
+/// CopyTarget is the target that can be used in `COPY INTO`.
+#[derive(Debug, Clone, PartialEq)]
+pub enum CopyTarget<'a> {
```

Comment:
I find some variants of this can be both `Source` and `Target`, what about renaming to `CopyUnit` or `CopyItem`?

Reply:
Let's use `CopyUnit`!